tests/L0/run_optimizers/test_lamb.py

```diff
@@ -281,6 +292,9 @@ def __init__(self, *args, **kwargs):
     def test_float(self):
         self.gen_single_type_test(param_type=torch.float)
 
+    def test_bfloat16(self):
+        self.gen_single_type_test(param_type=torch.bfloat16, iters=4)
```

Comment:
Could you try this way and check the other member methods still see `self.iters` of 7?

ref: https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUp

```suggestion
        self.iters = 4
        self.gen_single_type_test(param_type=torch.bfloat16)
```

Reply:
This works, the other member methods still use `self.iters=7` .